pkgs/development/compilers/elm/packages/release.nix

```diff
@@ -7,5 +7,6 @@
     elm-make = callPackage ./elm-make.nix { };
     elm-reactor = callPackage ./elm-reactor.nix { };
     elm-repl = callPackage ./elm-repl.nix { };
+    elm-format = callPackage ./elm-format.nix { };
```

Comment:
Maybe it would be better to leave the comment there and leave the file completely auto-generated? A short notice like:

This is not a core Elm package, and it's hosted on GitHub.
To update, run:
> `cabal2nix --revision refs/tags/foo http://github.com/avh4/elm-format > packages/elm-format.nix`
, where `foo` is a tag for a new version, for example "0.3.1-alpha".

would suffice. Then one would look at the comment, copy-paste the command into the terminal changing the tag and - done!

If you go that way, please also regenerate the package from this command, too, so that there is nothing not-auto-generated left in that file.